docs/dcp2_operating_procedures.rst

```diff
@@ -0,0 +1,253 @@
+
+.. role:: raw-html(raw)
+   :format: html
+.. |nn| replace:: :raw-html:`<blockquote>`
+.. |ne| replace:: :raw-html:`</blockquote>`
+
+
+
+
+==========================
+DCP/2 Operating Procedures
+==========================
+
+|nn| Text in a block quote (like this) is not normative. It was included
+for informational purposes only and does not bear on the implementation.
+|ne|
+
+.. contents::
+
+
+
+
+
+Changing the HCA metadata schema
+================================
+
+Changes to the HCA metadata schema, even seemingly minor ones, can have
+significant impact on the individual components of the DCP system. The DCP
+consortium therefore uses a review process that ensures all teams in the
+consortium approve of a schema change **before** it 
+
+- affects the master branch of the `metadata-schema`_ repository,
+
+- is published on `schema.humancellatlas.org`_, and
+
+- is used by metadata staged for import into the production instance of TDR
+
+.. _schema.humancellatlas.org: https://schema.humancellatlas.org/a
+
+.. _metadata-schema: https://github.com/HumanCellAtlas/metadata-schema
+
+.. _schema-test-data: https://github.com/HumanCellAtlas/schema-test-data
+
+.. _dcp2: https://github.com/HumanCellAtlas/dcp2
+
+.. _DCP/2 system design: dcp2_system_design.rst
+
+
+Sizing schema changes
+---------------------
+
+A *schema change* is a set of related modifications to one or more of the 
```

Comment:
It is subjective but I think I give two good rules of thumb as a guideline. The current CGM PR started off with various things mixed in that ideally shouldn't have been. This is a ultimately a risk that authors have to take and judge. If a reviewer gets a kitchen sink PR, they'll reject it. Same if they can't see the tree for the forest because the PR is too small. If a PR groups unrelated changes A and B, it's possible that A is rejected so neither A and B gets accepted. If each change were filed separately, at least B would have been accepted. Feel free to suggest concrete changes that reflect the inherent subjectivity.

>I also think given how lengthy and involved the update and review process is

It is no more involved than the frustrating stalemate we've been operating under for the past four years where EBI creates facts and UCSC has to either scramble to catch up or throw in the gauntlet. Lets give this a chance.